README.md

```diff
@@ -27,3 +27,21 @@ Start the service:
 Then open the page with your favorite browser:
 
     http://localhost:8080/validator.html
+
+# License
+
+
+Licensed under either of
+
+ * Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or
+   http://www.apache.org/licenses/LICENSE-2.0)
+ * MIT license ([LICENSE-MIT](LICENSE-MIT) or
+   http://opensource.org/licenses/MIT)
+
+at your option.
+
+### Contribution
+
+Unless you explicitly state otherwise, any contribution intentionally submitted
+for inclusion in the work by you, as defined in the Apache-2.0 license, shall
+be dual licensed as above, without any additional terms or conditions.
```

Comment:
I copied it from [spaceapi-rs](https://github.com/coredump-ch/spaceapi-rs/blob/master/README.md#contribution)...